src/pages/visualization-view/visualization-view.js

```diff
@@ -80,11 +88,24 @@ const VisualizationView = (props) => {
             </Tab>
           </span>
           <span className='action-area'>
-            <React.Fragment>
+            <React.Fragment >
+              <TabButton className={`button-${isAuthenticated ? 'enabled' : 'disabled'}`} onClick={showLoginPrompt}>
+                <div title='Saved Visualizations'>
+                  <a href='/user' className={`header-item link-${isAuthenticated ? 'enabled' : 'disabled'}`}>
+                    <img className={`folder-icon`} src={folderIcon} />
+                  </a>
```

Comment:
Should we use a react router `Link` element here instead of the raw `a`?  I think we won't have to do a full page reload if we use the `Link` element.

Reply:
Yeah, good point